### PR TITLE
Backport chdir keeping options

### PR DESCRIPTION
**Pull request by waiting-for-dev**
This backports 889ac7b to 0.5.x branch, because it caused a regression where it was not possible to render utf8 partials. Do you think it is a good idea to release a new version with it?

References #104 

**Important**: I have never tried to backport anything in GH. I have just branched from `0.5.3`, cherry-picked the commit and added a commit with update version/changelog. If there is a better way, please go ahead.

Thanks :)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# 0.5.4 / 2019-01-01
```
Comment (waiting-for-dev) [inline CHANGELOG.md]: I added current date, if it is accepted but released in another day, it has to be updated.

**Comment by timriley** [comment]
Thanks for this, @waiting-for-dev, I'm more than happy to do a bug fix release with this included.

Would you mind making the following changes to this PR first, though:

1. Change the CHANGELOG header simply to be `# Unreleased`
2. Don't change `version.rb`

I'll take care of changing both of those at once when I push the release.

**Comment by waiting-for-dev** [comment]
Sure! Forced a push with the changes.

Thanks @timriley !

**Comment by timriley** [comment]
Thanks @waiting-for-dev, that's great. I'm just leaving this open until I have the time to take care of the release (likely in the next few days).

**Comment by timriley** [comment]
@waiting-for-dev released as 0.5.4. Thank you!

**Comment by waiting-for-dev** [comment]
Thanks @timriley for your quick feedback :smile:

**Pull request merged**